jumanji/environments/routing/cvrp/env.py

```diff
@@ -96,33 +97,36 @@ class CVRP(Environment[State]):
 
     def __init__(
         self,
-        num_nodes: int = 20,
-        max_capacity: int = 30,
-        max_demand: int = 10,
+        generator: Optional[Generator] = None,
         reward_fn: Optional[RewardFn] = None,
         viewer: Optional[Viewer[State]] = None,
     ):
         """Instantiates a `CVRP` environment.
 
         Args:
-            num_nodes: number of city nodes in the environment. Defaults to 20.
-            max_capacity: maximum capacity of the vehicle. Defaults to 30.
-            max_demand: maximum demand of each node. Defaults to 10.
+            generator: `Generator` whose `__call__` instantiates an environment instance.
+                Default option is 'UniformGenerator' which defaults to a CVRP instance with 20
+                cities, a maximum vehicle capacity of 30, and a maximum city demand of 10.
```

Comment:
```suggestion
                The default option is 'UniformGenerator' which randomly generates
                CVRP instances with 20 cities sampled from a uniform distribution.
```

Reply:
Right, we probably want that.

Reply:
What about the following instead?
```suggestion
                The default option is 'UniformGenerator' which randomly generates
                CVRP instances with 20 cities sampled from a uniform distribution,
                a maximum vehicle capacity of 30, and a maximum city demand of 10.
```